hack/update-openapi-spec.sh

```diff
@@ -83,7 +83,7 @@ fi
 
 kube::log::status "Updating " ${OPENAPI_ROOT_DIR}
 
-curl -w "\n" -fs "${API_HOST}:${API_PORT}/swagger.json" > "${OPENAPI_ROOT_DIR}/swagger.json"
+curl -w "\n" -fs "${API_HOST}:${API_PORT}/openapi/v2" > "${OPENAPI_ROOT_DIR}/swagger.json"
```

Comment:
I realize that the default is supposed to be JSon, but should we specifically ask for the json version when we *want* json?

Reply:
not necessarily opposed, but having this test the implicit json response doesn't seem bad either